netif.pyx

```diff
@@ -42,7 +42,9 @@ from libc.string cimport strcpy, memset, memcpy
 from libc.stdlib cimport malloc, realloc, free
 
 
-CLONED_PREFIXES = ['lo', 'tun', 'tap', 'bridge', 'epair', 'carp', 'vlan', 'lagg']
+CLONED_PREFIXES = [
+    'lo', 'tun', 'tap', 'bridge', 'epair', 'carp', 'vlan', 'lagg' 'pflog', 'pfsync',
```

Comment:
Issues with comma `'lagg' 'pflog', 'pfsync',` :P

Reply:
Haha caught once again

Reply:
Happens to all of us ;)

Reply:
fixed